apps/web/src/pages/Pools/PoolAdvertisementPage/components/EducationRequirements.tsx

```diff
@@ -105,20 +109,35 @@ const EducationRequirements = () => {
       </span>
       <RequirementCard>
         <Heading level="h4" size="h6" data-h2-margin-top="base(0)">
-          {intl.formatMessage({
-            defaultMessage: "2-year post-secondary",
-            id: "ZIwaDE",
-            description: "Title for the education requirements",
-          })}
+          {isIAP
+            ? intl.formatMessage({
+                defaultMessage: "High school diploma or GED",
+                id: "qF/JU3",
+                description:
+                  "Title for the education requirements (Indigenous apprenticeship program)",
```

Comment:
I think we have moved away from using "Indigenous apprenticeship program" and towards IT Apprenticeship Program for Indigenous Peoples or IAP, see: https://github.com/GCTC-NTGC/gc-digital-talent/pull/6608

Reply:
Good to know, updated in [update references to IAP](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6668/commits/4e732b87bc99ba78d72feda67ad30219ae34ce05)